GitUI/MessageBoxes.cs

```diff
@@ -35,6 +35,12 @@ public class MessageBoxes : Translate
         private readonly TranslationString _shellNotFoundCaption = new TranslationString("Shell not found");
         private readonly TranslationString _shellNotFound = new TranslationString("The selected shell is not installed, or is not on your path.");
 
+        private readonly TranslationString _scriptExecutionFailed = new TranslationString("Failed to execute '{0}' script. Reason:\n\n{1}");
+        private readonly TranslationString _runShellFailed = new TranslationString("Failed to run '{0}' shell. Reason:\n\n{1}");
```

Comment:
I think the popup reads better with "Failed to run shell 'pwsh'. Reas...":
```suggestion
        private readonly TranslationString _runShellFailed = new TranslationString("Failed to run shell '{0}'. Reason:\n\n{1}");
```